CMakeLists.txt

```diff
@@ -61,5 +61,4 @@ else()
 endif()
 
 # Generate version header
-file(READ "src/base/version.h.in" versionHeader)
-file(WRITE "src/base/version.h" "${versionHeader}")
+configure_file("src/base/version.h.in" "${CMAKE_CURRENT_SOURCE_DIR}/src/base/version.h" @ONLY)
```

Comment:
why did you choose `@ONLY` instead of `COPYONLY`?

Reply:
I picked one that I envision that would be useful in the future (even if we don't need it currently).